libzhlgen/test/functions/mac/1.6.13/ShipBlueprint.zhl

```diff
@@ -0,0 +1,25 @@
+"554889e54157415653504989fe488d05e????????9890641f68678030000":
+cleanup __amd64 void ShipBlueprint::destructor(ShipBlueprint *this);
+/*	
+If the first doesn't work try these BUT KEEP THE NoHOOKS INTACT!!!
+
+
+".554889e54157415653504989fe498b4628498b4e304839c175??4889c8eb??bb????????31d2662e0f1f840000000000488b3cd04885ff74??493b":
+noHook __amd64 void AchievementTracker::DO_NOT_HOOK_1();
+554889e55de9":
```

Comment:
Previous signature was a `noHook` but this signature is a global match and not a relative (`.`) match, noHook might be unnecessary or this signature might be defined incorrectly.